SPECS/linux-firmware/linux-firmware.spec

```diff
@@ -36,17 +60,40 @@ cp iwlwifi-8000C-*.ucode %{buildroot}%{_firmwarepath}
 
 %files
 %defattr(-,root,root)
-%license WHENCE
```

Comment:
WHENCE file should be included for licensing reasons. Probably should be included in all subpackages.

Reply:
Included in all packages from linux-firmware.